auth/src/main/res/values/colors.xml

```diff
@@ -1,9 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="fui_colorPrimary">#3F51B5</color>
```

Comment:
See my [earlier comment](https://github.com/firebase/FirebaseUI-Android/pull/815#discussion_r128875877). Those colors are basically a lie and we aren't expecting them to ever be used. The dev should have to override them so that FirebaseUI is styled correctly. In theory, 99% of people won't have to do anything because they'll already have those colors defined since AppComat uses them.

If you want, we can keep the defined colors but it makes it less clear to the developer that they need to override them because they won't get a build error. Your call! 😀